tests/unit/test_binning_model.py

```diff
@@ -0,0 +1,40 @@
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.models.binning_model import BinningModelObject
+
+
+@responses.activate
+def test_binning_model():
+
+    records_body = [
+        {
+            "id": ["d8b7351d-24ce-49aa-8655-5b5809ab6bb8"],
+            "isActive": ["true"],
+            "clauseId": ["2e6c5f1b-ed49-40ab-8cbb-350aded25070"],
+            "similarityFunction": ["COSINE"],
+            "tokenizer": ["DEFAULT"],
+            "fieldName": ["surname"],
+            "threshold": ["0.75"],
+        }
+    ]
+
+    records_url = (
+        f"http://localhost:9100/api/versioned/v1/projects/1/binningModel/records"
+    )
+    responses.add(
+        responses.GET,
+        records_url,
+        body='{"id":["d8b7351d-24ce-49aa-8655-5b5809ab6bb8"],"isActive":["true"],'
+        '"clauseId":["2e6c5f1b-ed49-40ab-8cbb-350aded25070"],'
+        '"similarityFunction":["COSINE"],"tokenizer":["DEFAULT"],'
+        '"fieldName":["surname"],"threshold":["0.75"]}',
+    )
+
+    unify = Client(UsernamePasswordAuth("username", "password"))
+
+    binning_object = BinningModelObject(unify)
```

Comment:
This needs to go through the sequence:
```
project = unify.projects.by_resource_id("1")
binning_model = project.binning_model()
```

Reply:
Should I add a binning_model() method to our Project class?/What exactly should project.binning_model() do?